src/build_scripts/buildlib.cprnc

```diff
@@ -51,15 +51,15 @@ def buildlib(bldroot, installpath, case):
     run_bld_cmd_ensure_logging("{}/tools/configure --mpilib=mpi-serial --macros-format=CMake --machine={} --compiler={}".format(cimeroot, case.get_value("MACH"), compiler), logger, from_dir=bldroot)
 
     cmake_args = get_standard_cmake_args(case, shared_lib=True)
-
-    cmake_cmd = "CIMEROOT={cimeroot} . ./.env_mach_specific.sh && NETCDF=$(dirname $(dirname $(which nf-config))) CIMEROOT={cimeroot} cmake {cmake_args} -DMPILIB=mpi-serial -DDEBUG=FALSE -C Macros.cmake {cimeroot}/tools/cprnc -DCMAKE_PREFIX_PATH={dest_path} -DBLDROOT={bldroot}".\
+    os.environ["CIMEROOOT"] = cimeroot
```

Comment:
```suggestion
    os.environ["CIMEROOT"] = cimeroot
```
I think this is a typo and should be CIMEROOT with two O's.

Reply:
Good catch